<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PGJIAOP" failures="4" tests="12" name="ChromeTest" time="239.956" errors="0" timestamp="2024-01-21T10:15:10 IST">
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="6.714"/>
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="12.862"/>
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="35.958"/>
  <testcase classname="automation.test.TestCase1" name="enteringCredentials" time="17.555"/>
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="12.793">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="7.218">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="5.925">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="5.364">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="5.539">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="5.383">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="7.079">
    <skipped/>
  </testcase> <!-- notEnteringCredentials -->
  <testcase classname="automation.test.TestCase1" name="notEnteringCredentials" time="8.59">
    <failure type="java.lang.AssertionError" message="Element not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element not found expected [true] but found [false]
at pageEvents.LoginPageEvents.verfiyIfLoginPageNotLoaded(LoginPageEvents.java:14)
at automation.test.TestCase1.notEnteringCredentials(TestCase1.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- notEnteringCredentials -->
</testsuite> <!-- ChromeTest -->
